packages/styles/components/_c.fields.scss

```diff
@@ -0,0 +1,37 @@
+.mc-field {
+  &__label {
+    @include set-font-scale('04','xs');
+
+    color: $color-fields-label;
+
+    &-wrapper {
+      margin-bottom: $mu050;
+    }
+  }
+
+  &__error-message {
+    @include set-font-scale('04','xs');
+
+    color: $color-fields-error;
+    margin-top: $mu025;
+  }
+
+  &__required {
+    @include set-font-scale('02','xs');
+
+    color: $color-fields-mandatory;
+
+    &::before {
+      content: "-";
+      margin-right: $mu025;
```

Comment:
If I look at the design files, this margins seems too big.
maybe just : 

```scss
content: " - ";
// or with non-breaking spaces : 
content: "\00a0-\00a0";
```

Reply:
fixed